askametric/validation/validation_prompts.py

```diff
@@ -0,0 +1,94 @@
+# Prompts for the validation pipeline
+
+
+def grading_bot_prompt():
+    """
+    System prompt for the grading bot
+    """
+    prompt = """
+    You are a grading bot. You will be asked to evaluate messages that
+    you receive, based on some criteria.
+    You should give a grade of 1 if the message satisfies the given
+    criteria, and 0 if it does not.
+
+    You should respond in the following json format:
+    {"score": your grade, "reason": reason for your grading}
+    """
+    return prompt
+
+
+def get_relevancy_prompt(question: str, llm_response: str):
+    """
+    Create prompt to check relevancy of generated response.
+    """
+    prompt = f"""
+    Here is the message to evaluate
+    ---- Message Begins----------------
+    Question: {question}
+    Answer: {llm_response}
+    ---- Message Ends------------------
+
+    ---- Evaluation Criteria ----
+    Does "Answer" address the key elements of the "Question"?
+    """
+    return prompt
+
+
+def get_accuracy_prompt(correct_answer: str, llm_response: str):
+    """
+    Create prompt to check accuracy of generated response.
+    """
+    prompt = f"""
+    Here is the message to evaluate
+    ----Message Begins----------------
+    Correct Answer: {correct_answer}
+    Answer: {llm_response}
+    ----Message Ends----------------
+
+    ---- Evaluation Criteria ----
+    Does the "Answer" match "Correct Answer" as CLOSELY as possible?
+    Remember, the "Answer" and "Correct Answer" MUST have the same
+    content and be similar in semantic meaning. If "Answer" has additional
```

Comment:
This was failing for me for the "Apka prashn surakshit nhi hai kyunki isme PII Hai" on the accuracy part. So, my accuracy result on the 2 questions was 50% but sometimes on re-runs I got 100%.

So, I think this prompt is a bit too strict.